src/dictionaries.cpp

```diff
@@ -85,6 +85,13 @@ Dictionaries::Dictionaries(const std::string& filePath)
                 vendorName = tokens[1];
             else if (tokens[0] == "END-VENDOR")
                 vendorName.clear();
+            else if (tokens[0] == "$INCLUDE")
+            {
+                if (tokens[1].substr(0, 1) == "/")
+                    Dictionaries fillingDictionaries(tokens[1]);
+                else
+                    Dictionaries fillingDictionaries(filePath.substr(0, filePath.rfind('/') + 1)+ tokens[1]);
```

Comment:
Okay, you parsed the file in a local dictionary. Now what? Everything you parsed there is lost, the variable gets destroyed when it leaves the scope.

What we need here is to take all that data and put it into our local dictionaries.